authorization_django/middleware.py

```diff
@@ -166,11 +166,14 @@ def get_claims(jwt):
                 'scopes': {convert_scope(r) for r in claims['realm_access']['roles']},
                 'claims': claims
             }
-        elif claims.get('scp') and claims.get('preferred_username'):
-            # Microsoft token structure
+        elif claims.get('groups') and (claims.get('unique_name') or claims.get('un')):
```

Comment:
Ik zou hier extra haakjes omheen zetten om duidelijker te maken hoe de condities gegropeerd zijn.

Reply:
Daar staan ze, toch? Of bedoel je iets anders?

Reply:
:man_facepalming: